src/pages/contributors/herve-le-meur.adoc

```diff
@@ -0,0 +1,76 @@
+
+In my head I was a programmer, not a sysadmin.
+So when link:https://www.jenkins.io/blog/authors/markewaite/[Mark Waite] was interviewing me and asked me some network questions, this was a bit daunting and seemed potentially impossible.
+However, I had an interview with Mark, link:https://www.jenkins.io/blog/authors/dduportal/[Damien Duportal], and link:https://www.jenkins.io/blog/authors/olblak/[Olivier Vernin] that was a fantastic experience.
+We were able to have a meaningful discussion that helped me feel more comfortable, and made my decision easier.
+15 minutes before my interview, I got a final offer from another company (where Damien & link:https://www.jenkins.io/blog/authors/jmmeessen/[Jean-Marc Meessen] had previously worked), hesitated and it turned out for the best since I'm still part of the Jenkins project today.
+I also had experience moderating an online forum, so the community part of the role was familiar to me.
+
+== How long have you been using Jenkins?
+
+I started with Jenkins-x but had never touched Jenkins proper.
+My initial impression of Jenkins was not positive, thinking it was due to how bad and unstable it was.
+However, once I started using Jenkins, it was the comparrison was night and day.
+My first impression was that it was heavy & slow compared to others.
+I was not alone in thinking that Jenkins is not necessarily the best, fastest, or newest project, but was proven wrong once I started working on the project.
+
+== Why choose Jenkins over other projects?
+
+I didn't necessarily choose Jenkins as much as it is the main part of my job.
+As part of my role, I've spent time looking at what Olivier, Damien & Mark have put in place for infrastructure.
+Jenkins is much more improved and efficient, with complete integration from server to image delivery, we have control and segmentation over every aspect.
+I love the fact that we are using Jenkins to develop and distribute Jenkins.
+This is unique because most open-source tools don't have transference success, where Jenkins has spent so much time and effort building to work with developer processes and functionalities.
+In my eyes, this is one of the main aspects of why Jenkins is successful.
+Jenkins has also integrated other tools, such as Terraform, Puppet, academage, and M-Files with Jenkins, but Jenkins is the main orchestrator of these.
+Other projects and tools do not own their issues, but Jenkins owns, and works on its issues, in order to improve.
+I've always liked creating and working on tools, so working on Jenkins is a crowning achievement for me, even if it's not core.
+
+== What growth have you seen in Jenkins since joining the project?
+
+We have been getting more and more instances defined as code, this means that they do not have to go in and configure settings manually since the code is pre-defined.
```

Comment:
```suggestion
We have been getting more and more instances defined as code.
This means that users do not have to go in and configure settings manually since the code is pre-defined.
```